### PR TITLE
feat(Sanitizer): added flex properties

### PR DESCRIPTION
**Pull request by raiseandfall**
- added `flex-direction` & `align-items` to whitelist

### CODE REVIEW

**Comment by lemoustachiste** [comment]
potentially we could see with rollup to add a text at the top of the sanitizer indicating this, as well as using `terse` for the `sanitizer.js` file so it looks like an unattractive file to modify. If you could do it with this PR, it'd be grand.

I see you've done it before though: https://github.com/blockchain-certificates/blockcerts-verifier/pull/125/files, so you might need to address this in this PR.

File: README.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ Will make the demo page available on http://localhost:8081/demo/.
 The `sanitizer` is used in order to protect against malicious certificates that could hold XSS attacks.
 It is an overlay of the [xss](https://www.npmjs.com/package/xss) library, since at times, you might want to be able to configure or adapt the whitelist to your own needs.
 
-To modify it, you should modify the index.js file. Then run:
+To modify it, you should modify the index.js file by adding the CSS properties to whitelist to the object `whiteListedCssProperties`, then run:
```
Comment (lemoustachiste) [inline README.md]: I think the CSS part should be a separate paragraph, you can modify index.js for other purposes too

File: rollup-sanitizer.config.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
 export default {
   input: 'sanitizer/index.js',
   name: 'Sanitizer',
   output: [
     {
       file: 'sanitizer/sanitizer.js',
-      format: 'es'
+      format: 'es',
+      banner: '/**\n * @warning\n *\n * THIS IS AN AUTO GENERATED FILE. IF YOU WISH TO WHITELIST PROPERTIES / TAGS, PLEASE DO IT SO IN sanitizer/index.js \n *\n * More Information: https://github.com/blockchain-certificates/blockcerts-verifier#modifying-the-sanitizer\n **/\n'
```
Comment (lemoustachiste) [inline rollup-sanitizer.config.js]: 👍 excellent

File: README.md (hunk 1 above)
Comment (raiseandfall) [reply README.md]: Good point

**Pull request merged**